plugins/discourse-ai/lib/completions/dialects/chat_gpt.rb

```diff
@@ -69,7 +69,10 @@ def tools_dialect
         # developer messages are preferred on recent reasoning models
         def supports_developer_messages?
           !legacy_reasoning_model? && llm_model.provider == "open_ai" &&
-            (llm_model.name.start_with?("o1") || llm_model.name.start_with?("o3"))
+            (
+              llm_model.name.start_with?("o1") || llm_model.name.start_with?("o3") ||
+                llm_model.name.start_with?("gpt-5")
+            )
```

Comment:
it would be nice to have `REASONING_MODEL_PREFIX_LIST`. Just ultra NIT improvement